src/map/status.cpp

```diff
@@ -8838,17 +8878,20 @@ t_tick status_get_sc_def(struct block_list *src, struct block_list *bl, enum sc_
 			tick_def2 = (status->vit + status->dex) * 50;
 			break;
 		case SC_WHITEIMPRISON:
-			if( tick == 5000 ) // 100% on caster
+			if( src == bl ) // 100% on caster
 				break;
-			if( bl->type == BL_PC )
-				tick_def2 = status_get_lv(bl)*20 + status->vit*25 + status->agi*10;
-			else
-				tick_def2 = (status->vit + status->luk)*50;
+			sc_def = status->str * 10 + status_get_lv(bl) * 10 + status->luk * 10;
+			tick_def2 = -2000;
+			break;
+		case SC_FEAR:
+			sc_def = status->int_ * 10 + status_get_lv(bl) * 10 + status->luk * 10;
+			tick_def2 = -2000;
 			break;
 		case SC_BURNING:
-			tick_def2 = 75*status->luk + 125*status->agi;
+			sc_def = status->agi * 10 + status_get_lv(bl) * 10 + status->luk * 10;
+			tick_def2 = -2000;
 			break;
-		case SC_FREEZING:
+		case SC_FREEZING: // TODO: Confirm
```

Comment:
Please also check the minimum defined below (very end of this function), it still says it needs to be tested.